docker/docker-compose.objects-apis.yml

```diff
@@ -1,11 +1,15 @@
 version: '3'
```

Comment:
I get the warning that `version` parameter is obsolete, but running with the old `docker-compose` complains about the `name`.

Reply:
I'd use `version: '3.8'`, 3 is very old and equivalent to 3.0

Reply:
`docker-compose` is probably too old to support the `name` attribute, but I guess `docker compose` should be used anyway.

Regarding the `version` attribute, it seems to be there for [backwards compatibility](https://docs.docker.com/compose/compose-file/04-version-and-name/#version-top-level-element-optional). The compose format is now generic and not tied to docker anymore apparently. Maybe in the future we should update all our docker compose files to remove this version, but we would have to make sure everyone is on the latest docker engine version.

Reply:
my experience is that with docker-compose versions you need to be very conservative. I remember using a version that was released more than 1+ year ago and had to revert to an older one because people did not have those runtimes on their machines yet and didn't have admin rights to update.